hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/OzoneAddress.java

```diff
@@ -85,7 +89,34 @@ public OzoneAddress(String address)
 
   }
 
-  public OzoneClient createClient(OzoneConfiguration conf)
+  @VisibleForTesting
+  protected OzoneClient createRpcClient(
```

Comment:
protected OzoneClient createRpcClient(ConfigurationSource conf) fits in a single line